clients/bigquery/dialect/dialect.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	stagingAlias = "cc"
```

Comment:
Can we move this to a shared constant vs. pkg level?

Reply:
Yep.